modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1700,13 +1700,13 @@
     <string name="onboarding_patron_feature_gratitude_title" translatable="false">@string/onboarding_plus_feature_gratitude_title</string>
     <string name="onboarding_patron_subscribe">Subscribe to Pocket&#160;Casts Patron</string>
     <string name="onboarding_plus_features_title" translatable="false">@string/onboarding_upgrade_everything_you_love_about_pocket_casts_plus</string>
-    <string name="onboarding_plus_feature_desktop_apps_title">Desktop Apps</string>
+    <string name="onboarding_plus_feature_desktop_apps_title">Desktop Apps &amp; web apps</string>
```

Comment:
Should we rename `onboarding_plus_feature_desktop_apps_title` to `onboarding_plus_feature_desktop_and_web_apps_title`?

Reply:
Sure! I've made a https://github.com/Automattic/pocket-casts-android/pull/1702 to address this changes.